mycity/mycity/utilities/gis_utils.py

```diff
@@ -7,8 +7,12 @@
 """
 
 from arcgis.features import FeatureLayer
-
 import mycity.utilities.google_maps_utils as g_maps_utils
+from arcgis.gis import *
```

Comment:
Can we just import what we need from the library, or do `import arcgis.gis as gis` and access the members we need from that?